packages/cli/tests/e2e/bot/CanDeployToCustomizedRg.tests.ts

```diff
@@ -63,13 +63,9 @@ describe("Deploy to customized resource group", function () {
     {
       const context = await readContextMultiEnv(projectPath, env);
 
-      // Validate Aad App
```

Comment:
Also remove AadValidator and AppStudioLogin from import since they are of no use.

Reply:
Fixed, thx！